src/index.html

```diff
@@ -7,6 +7,310 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <a
+          class="logo__link"
+          href="#home"
+        >
+          <img
+            class="logo__img"
+            src="images/logo.png"
+            alt="logo"
+          >
+        </a>
+      </div>
+      <nav class="menu">
+        <ul class="menu__items">
+          <li class="menu__item">
+            <a
+              class="menu__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="https://samsung.com"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="#"
+            >
```

Comment:
```suggestion
            <a class="menu__link" href="#">
```
Если у тега меньше трех атрибутов - можно записывать вот так